### PR TITLE
[port_util] add get_rif_port_map, get_vlan_interface_oid_map

### PR DESCRIPTION
**Pull request by mykolaf**
Add 2 new utility functions, to be used by snmpagent.

Related to [#133](https://github.com/Azure/sonic-snmpagent/pull/133)

### CODE REVIEW

File: src/swsssdk/port_util.py [hunk 1]
```diff
@@ -11,12 +11,14 @@
...
 """
 SONIC_ETHERNET_BP_RE_PATTERN = "^Ethernet-BP(\d+)$"
+SONIC_VLAN_RE_PATTERN = "^Vlan(\d+)$"
 SONIC_PORTCHANNEL_RE_PATTERN = "^PortChannel(\d+)$"
 SONIC_MGMT_PORT_RE_PATTERN = "^eth(\d+)$"
 
 
 class BaseIdx:
     ethernet_base_idx = 1
+    vlan_interface_base_idx = 2000
```
Comment (qiluo-msft) [inline src/swsssdk/port_util.py:21]: @prsunny @judyjoseph do you think it leaves enough indexes to portchannels in multiple-ASIC case?

File: src/swsssdk/port_util.py [hunk 2]
```diff
@@ -103,3 +108,40 @@ def get_vlan_id_from_bvid(db, bvid):
...
+
+    return rif_port_oid_map
+
+def get_vlan_interface_oid_map(db):
+    """
+        Get Vlan Interface names and sai oids
+    """
+    db.connect('COUNTERS_DB')
+    rif_name_map = db.get_all('COUNTERS_DB', 'COUNTERS_RIF_NAME_MAP', blocking=True)
+    rif_type_name_map = db.get_all('COUNTERS_DB', 'COUNTERS_RIF_TYPE_MAP', blocking=True)
```
Comment (qiluo-msft) [inline src/swsssdk/port_util.py:136]: also check None?

File: src/swsssdk/port_util.py [hunk 3]
```diff
@@ -103,3 +108,40 @@ def get_vlan_id_from_bvid(db, bvid):
...
+
+    if not rif_name_map:
+        return {}
+
+    oid_pfx = len("oid:0x")
+    vlan_if_name_map = {sai_oid[oid_pfx:]: if_name for if_name, sai_oid in rif_name_map.items()
```
Comment (qiluo-msft) [inline src/swsssdk/port_util.py:142]: This statement is too long to understand. Could you refactor to a loop with if-block? And add some comment to explain?

File: src/swsssdk/port_util.py (hunk 1 above)
Comment (judyjoseph) [reply src/swsssdk/port_util.py:21]: I feel 1000 port channel interfaces should be good enough even for Chassis. But I am looking to see why the vlan_interface_base_idx is defined  .. is it for this API get_vlan_interface_oid_map() ?

Comment (prsunny) [reply src/swsssdk/port_util.py:21]: I think this just simply leaves a gap from 6000-9000 after `2000+4094`, why not start from 4000?

File: src/swsssdk/port_util.py (hunk 3 above)
Comment (mykolaf) [reply src/swsssdk/port_util.py]: Sure

File: src/swsssdk/port_util.py (hunk 1 above)
Comment (mykolaf) [reply src/swsssdk/port_util.py:21]: 2K BaseIdx is what our Arch and Guohan agreed on. If there is any concern about the size of pool, should discuss with them.

Comment (mykolaf) [reply src/swsssdk/port_util.py:21]: @judyjoseph 
>  is it for this API get_vlan_interface_oid_map() ?
Theoretically index can be used anywhere where a numeric identification for interface is needed. 
In my case I am using it for interface ID in the SNMP Interface MIB (RFC1213).

Comment (judyjoseph) [reply src/swsssdk/port_util.py:21]: lgtm .. I see that the index range discussion already happened.

**Pull request merged**